posts/2021-09-17-binding-app-to-postgresql-db-with-odo-and-ol-stack.adoc

```diff
@@ -1,40 +1,43 @@
 ---
 layout: post
-title: "Bind an Operator Managed PostgreSQL Database to a JPA Application in a Kubernetes Cluster"
+title: "Bind an operator-managed PostgreSQL database to a JPA application in a Kubernetes cluster"
 categories: blog
 author_picture: https://avatars3.githubusercontent.com/mezarin
 author_github: https://github.com/mezarin
-seo-title: Bind an Operator Managed PostgreSQL Database to a JPA Application in a Kubernetes Cluster - OpenLiberty.io
-seo-description: Speed up development by making it easier for application developers to bind operator managed services to applications.
-blog_description: "Speed up development by making it easier for application developers to bind operator managed services to applications"
+seo-title: Bind an operator-managed PostgreSQL database to a JPA application in a Kubernetes cluster - OpenLiberty.io
+seo-description: Speed up development by making it easier for application developers to bind operator-managed services to applications.
+blog_description: "Speed up development by making it easier for application developers to bind operator-managed services to applications"
 open-graph-image: https://openliberty.io/img/blog/ol-stack-jpa-app-db-bind-browser-data-entry.png
 ---
-= Bind an Operator Managed PostgreSQL Database to a JPA Application in a Kubernetes Cluster
+= Bind an operator-managed PostgreSQL database to a JPA application in a Kubernetes cluster
 Edward Mezarina <https://github.com/mezarin>
 
-Binding your Microservice application to operator based services (e.g. databases, etc.) has now been made easier through automatic collection of service data, deployment of needed resources, and data sharing with the application.
+Binding your microservice application to operator-based services , such as a database, is now easier, thanks to the Service Binding Operator and OpenShift Do (odo). These frameworks enable the automatic collection of service data and easier resource data sharing while abstracting resource configuration and deployment details.
```

Comment:
add a comma before "while"- _unless_ the "while" clause applies only to "resource data sharing"

These frameworks enable the automatic collection of service data and easier resource data sharing, while abstracting resource configuration and deployment details.